blank-canvas-blocks/experimental-theme.json

```diff
@@ -85,6 +85,14 @@
 				]
 			},
 			"custom": {
+				"fonts": {
+					"primary": "'Playfair Display', Georgia, Times, serif",
+					"secondary": "'Fira Sans', Helvetica, Arial, sans-serif"
+				},
```

Comment:
What is the advantage of using "custom.fonts.primary" over the font preset variables?

Reply:
It's just for testing purposes.